config/local/api-defs-http/onboarding-sample.yml-sample

```diff
@@ -0,0 +1,22 @@
+services:
```

Comment:
Why do you use such a strange name?

Reply:
that was a copy from original apidefs, I removed this definition